app/src/androidTest/java/com/sdp/movemeet/view/profile/DrawableMatcher.java

```diff
@@ -0,0 +1,68 @@
+package com.sdp.movemeet.view.profile;
+
+import android.content.res.Resources;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.drawable.Drawable;
+import android.view.View;
+import android.widget.ImageView;
+
+import org.hamcrest.Description;
+import org.hamcrest.TypeSafeMatcher;
+
+public class DrawableMatcher extends TypeSafeMatcher<View> {
```

Comment:
Would you mind documenting this class?

Reply:
Yes probably! This is a class I added to be able to assert that the picked image corresponds to the uploaded image in EditProfileActivityImagePickerTest. Initially I was not sure this mechanism would work and since I implemented it tonight (I have found it on the internet), I haven't taken the time to document it yet! But I will definitely do it!